grafast/website/grafast/step-library/standard-steps/list.md

```diff
@@ -8,3 +8,9 @@ Usage:
 ```ts
 const $abTuple = list([$a, $b]);
```

Comment:
```suggestion
const $abTuple = list([$a, $b, $c]);
```

Reply:
is it still a tuple?

Reply:
[Indeed](https://en.wikipedia.org/wiki/Tuple). To be more specific we could call it a 3-tuple (or n-tuple).

Reply:
But it should probably be `$abcTuple` now.